### PR TITLE
feat(whitepaper): add tutorials & Tech doc cards

### PR DESCRIPTION
**Pull request by houssemmimoun27**
This PR add the Tutorials and Technical documentation cards to the home page.

<details>

<summary> Screenshots </summary>

Screenshot 2023-08-03 at 15 08 46

</details>

### CODE REVIEW

**Review by MdechampG — CHANGES_REQUESTED** [review]
Thanks @houssemmimoun27 
Could you keep the same gap between the 2 rows as here : 

image

File: docs/index.mdx [hunk 1]
```diff
@@ -26,6 +26,26 @@ Welcome! OKP4 is a public PoS layer 1 blockchain built for trust-minimized data
         </a>
         <p class="card-text">Everything that an OKP4 validator should know.</p>
       </div>
+    </div>
+     <div class="col">
+      <div class="card card-body h-100 d-flex flex-column">
+        <a href="/technical-documentation/introduction" class="card-title card-link stretched-link">
+          <h2>Technikal Documentation</h2>
```
Comment (ccamel) [inline docs/index.mdx:33]:
```suggestion
          <h2>Technical Documentation</h2>

```

**Review by ccamel — APPROVED** [review]
LGTM

File: docs/index.mdx [hunk 2]
```diff
@@ -26,6 +26,26 @@ Welcome! OKP4 is a public PoS layer 1 blockchain built for trust-minimized data
         </a>
         <p class="card-text">Everything that an OKP4 validator should know.</p>
       </div>
+    </div>
+     <div class="col">
+      <div class="card card-body h-100 d-flex flex-column">
+        <a href="/technical-documentation/introduction" class="card-title card-link stretched-link">
+          <h2>Technical Documentation</h2>
+        </a>
+        <p class="card-text">Deep dive into the OKP4 protocol.</p>
```
Comment (fredericvilcot) [inline docs/index.mdx:35]: To be compliant with content at line 47, I'll recommend to capitalize the `protocol` word:
```suggestion
        <p class="card-text">Deep dive into the OKP4 Protocol.</p>

```

**Review by fredericvilcot — CHANGES_REQUESTED** [review]

**Review by fredericvilcot — APPROVED** [review]
👍

**Pull request merged**